composer.json

```diff
@@ -62,6 +62,7 @@
         "ezsystems/date-based-publisher": "^1.0@dev",
         "ezsystems/content-on-the-fly-prototype": "~0.1.7",
         "ezsystems/ezplatform-multi-file-upload": "^0.1.0@dev",
+        "ezsystems/ezplatform-http-cache-fastly": "dev-fastly-driver",
```

Comment:
should be updated before merge I guess

Reply:
Yes, but I don't know what we'll call the version yet
1.0.0-beta1?